API/Data/Metadata/ComicInfo.cs

```diff
@@ -125,4 +125,18 @@ public static void CleanComicInfo(ComicInfo info)
     }
 
 
+    // TODO: Example of Equals() implementation. This currently only
```

Comment:
Please finish out the implementation for Equals, implementing a half implementation won't get this PR merged as it opens potential bugs.